semanticdb-kotlinc/src/test/kotlin/com/sourcegraph/semanticdb_kotlinc/test/SemanticdbSymbolsTest.kt

```diff
@@ -590,4 +592,44 @@ class SemanticdbSymbolsTest {
                                         }
                                     }))))
             .mapCheckExpectedSymbols()
+
+    @TestFactory
+    fun kdoc() =
+        listOf(
+            ExpectedSymbols(
+                "empty kdoc line",
+                SourceFile.testKt(
+                    """
+                    |/**
+                    |
```

Comment:
To make the test cover the case I mentioned in my previous comment, how about we put an empty line **amidst** non-empty ones, to ensure that all the lines of kdoc are collected?

Reply:
Is there a difference between whats here and your recommendation? Not sure if Im missing what youre thinking here, from what Im getting it seems like this test case should be almost identical in that with the previous code, itd return empty string as it hits the empty line and terminates